spec.html

```diff
@@ -10730,10 +11114,40 @@ <h1>ECMAScript Function Objects</h1>
         </tr>
         <tr>
           <td>
-            [[IsClassConstructor]]
+            [[Fields]]
           </td>
           <td>
-            Boolean
+            List of ClassFieldDefinition Records
+          </td>
+          <td>
+            If the function is a class, this is a list of Records representing the non-static fields and corresponding initializers of the class.
+          </td>
+        </tr>
+        <tr>
+          <td>
+            [[PrivateMethods]]
+          </td>
+          <td>
+            List of PrivateElement Records
+          </td>
+          <td>
+            If the function is a class, this is a list representing the non-static private methods and accessors of the class.
+          </td>
+        </tr>
+        <tr>
+          <td>
+            [[ClassFieldInitializerName]]
+          </td>
+          <td>
+            String | Symbol | Private Name | ~empty~
```

Comment:
What happens when [[ClassFieldInitializerName]] is a Private Name? NamedEvaluation can't expose a PrivateName.

Reply:
I'm not sure what you mean by "expose". NamedEvaluation calls InstantiateOrdinaryFunctionExpression (or variants), which calls SetFunctionName, which has

```
3. Else if _name_ is a Private Name, then
  a. Set _name_ to _name_.[[Description]].
```

before installing _name_ to the function object. That doesn't expose the Private Name itself, but does expose its string representation. Is that your concern?

Reply:
Ah yes, that's what I was wondering and I had missed that, thanks.